pulsar-io/kafka/src/main/java/org/apache/pulsar/io/kafka/KafkaSinkConfig.java

```diff
@@ -81,6 +81,12 @@
             "The serializer class for Kafka producer to serialize values. You typically shouldn't care this. "
           + "Since the serializer will be set by a specific implementation of `KafkaAbstractSink`.")
     private String valueSerializerClass = "org.apache.kafka.common.serialization.ByteArraySerializer";
+    @FieldDoc(
+        defaultValue = "",
+        help =
+            "The producer config properties in the form \"key1:val1,key2:val2\" to be passed to Producer. "
+                + "Note that other properties specified in the connector config file take precedence over this config.")
+    private String producerConfigProperties;
```

Comment:
@murong00 connector config is a yaml file. I think you can use Map<String, Object> for properties, that's probably better than using a single string for multiple key/value settings.

Reply:
Changed, thanks!